doc/content/devices/best-practices/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Best Practices"
+description: ""
+weight:
+---
+
+This section contains best practices for building connected devices which use {{% tts %}} as a Network Server.
+
+<!--more-->
+
+LoRaWAN devices should always comply to the [LoRaWAN specification](https://www.lora-alliance.org/lorawan-for-developers), with special regard to minimizing join requests and duty cycle limitations.
+
+## Minimize Join Requests
+
+The LoRaWAN specification warns especially against systematic rejoin in case of network failure. A device should keep the result of an activation in permanent storage if the device is expected to be turned off during its lifetime.
+
+A device may temporarily lose connection with the network for many reasons: if the network server or gateways are suffering from an outage, if there’s no coverage in the area, etc. Whenever a device rejoins, it consumes the closest gateway’s airtime to emit the downlink - and if many devices in an area are rejoining at the same time (e.g. in case of a temporary gateway outage), this leads to network bloating in the area.
+
+## Maximum Duty Cycle
+
+The duty cycle of radio devices is often regulated by government. If this is the case, the duty cycle is commonly set to 1%, but make sure to check the regulations of your local government to be sure.
+
+In Europe, duty cycles are regulated by section 7.2.3 of the ETSI EN300.220 standard. This standard defines the following sub-bands and their duty cycles:
+
+- g (863.0 – 868.0 MHz): 1%
+- g1 (868.0 – 868.6 MHz): 1%
+- g2 (868.7 – 869.2 MHz): 0.1%
+- g3 (869.4 – 869.65 MHz): 10%
+- g4 (869.7 – 870.0 MHz): 1%
```

Comment:
I'm quite sure this is outdated, there's a new spec.

Reply:
Also this gets too much attention here. I don't think it adds much, just refer to the spec and mention the regions it applies to.